canonical/snapcraft.io
# Update dependencies version

## Done
- Update store-api
- Update store-base
## How to QA
- Go to https://snapcraft-io-4484.demos.haus/beta-store and check that the page returns 100 snaps, then click on categories and ensure that the snaps are filtered accordingly returning 100 snaps or less depending on the categories
- Go to https://snapcraft-io-4484.demos.haus/beta/store.json?architecture=arm64 or any other architecture and ensure it returns snaps with that architecture (snap results don't have the architecture key though)
## Issue / Card https://warthogs.atlassian.net/browse/WD-7483
Fixes #

## Screenshots
